spec/factories/claim/advocate_claims.rb

```diff
@@ -91,24 +83,18 @@
       after(:create) { |claim| allocate_claim(claim); claim.reload }
     end
 
-    trait :allocated do
-      after(:create) { |claim| allocate_claim(claim); claim.reload }
-    end
-
+    # DEPRECATED see shared traits
```

Comment:
:+1: for the comment. I was unaware that these factories were deprecated. Thanks.

Reply:
👌  They can, and are, still used but are a duplication of logic that is/should-be generic to all claims. I only left them here to avoid the yak-shaving involved in remove them.

We should remove them when time permits.